smartseq2_single_sample/pipelines/hisat2_rsem_pipeline.wdl

```diff
@@ -31,7 +31,7 @@ workflow RunHisat2RsemPipeline {
       ref_name = hisat2_ref_trans_name,
       sample_name = sample_name,
       output_name = output_prefix,
-      disk_size = fastq_size*md_disk_multiplier+hisat2_ref_size+additional_disk
+      disk_size = fastq_size*md_disk_multiplier+hisat2_ref_size+additional_disk*5.0
```

Comment:
Same as above. Also, is disk size specified in GB by default? I thought it was bytes, in which case this may not function well with the `size` calls above. 

... the wdl draft 1 and 2 specs don't seem to comment on `disk_size` at all, so I don't know the answer for sure. You could check in a metadata file.